File: Baekjoon/BaaarkingDog/0x02_basic/2490.cpp
// 2490: 윷놀이

#include <iostream>

int main() {
  for (int i=0; i<3; i++) {
    int n1, n2, n3, n4, sum = 0;
    std::cin >> n1 >> n2 >> n3 >> n4;

    sum += n1 + n2 + n3 + n4;

    if (sum == 0) std::cout << 'D';
    else if (sum == 1) std::cout << 'C';
    else if (sum == 2) std::cout << 'B';
    else if (sum == 3) std::cout << 'A';
    else if (sum == 4) std::cout << 'E';
    std::cout << std::endl;
  }

  return 0;
}

File: Baekjoon/BaaarkingDog/0x02_basic/2490.py
# 2490: 윷놀이

for _ in range(3):
  input_nums = list(map(int, input().split()))

  result = sum(input_nums)
  if (result == 0): print("D")
  elif (result == 1): print("C")
  elif (result == 2): print("B")
  elif (result == 3): print("A")
  else: print("E")

